package.json

```diff
@@ -19,7 +19,8 @@
   },
   "license": "MIT",
   "author": "Robert Jackson <me@rwjblue.com>",
-  "main": "lib/index.js",
+  "exports": "./lib/index.js",
```

Comment:
I think we will need to expose `lib/helpers/rule-test-harness.js` as well. It is used by the plugins to test their rules ([example](https://github.com/ember-template-lint/ember-template-lint-plugin-prettier/blob/e5074af45ddbc7ee42e76d244a4e3d556a755f77/test/unit/rules/lint-prettier-test.js#L1)).

But maybe it is the good time to expose it through `lib/index.js`? Wdyt?

Reply:
Good point, my preference would be that we expose that through `lib/index.js`, as I've implemented here: https://github.com/ember-template-lint/ember-template-lint/pull/2209